Loop/Managers/DeviceDataManager.swift

```diff
@@ -1372,156 +1388,144 @@ extension DeviceDataManager {
             }
         }
         
-        let command: RemoteCommand
+        let action: RemoteAction
         
```

Comment:
RemoteCommand was renamed to RemoteAction. Later on I plan to reintroduce a RemoteCommand, but it will instead _have_ a RemoteAction. The distinction is a command will include the action and all the ancillary things related to commands (security validation, status reporting, etc). 

Separating these concepts should make more sense with Remote 2.0.